source/crud/tests/unified/findOneAndDelete-let.yml

```
@@ -100,8 +65,8 @@ tests:
           let: &let1
             id: 1
         expectError:
-          # Older server versions may not report an error code, but the error
-          # message is consistent between 2.6.x and 4.4.x server versions.
+          # This error message is consistent between for 4.4+ servers.
```

Did you mean this?

```suggestion
          # This error message is consistent between 4.2.x and 4.4.x servers.
```

Perhaps we don't need a comment here at all. I noticed you didn't do so for the update and delete tests, which all require 3.6+ (a wider range of server versions).

Whatever you replace here, please do the same for the findOneAndReplace and findOneAndUpdate tests below.